api/database/factories/PoolFactory.php

```diff
@@ -72,57 +71,93 @@ public function definition()
         ];
     }
 
-    public function configure()
```

Comment:
The removal of the `configure()` function seemed to trip some stuff up as I believe it runs automatically.
Resolved test failure by calling the needed function in https://github.com/GCTC-NTGC/gc-digital-talent/pull/10290/commits/8f825e6c5277fba0928313796d0842b4fa771c53